decksite/controllers/api.py

```diff
@@ -143,6 +157,41 @@ def decks_api() -> Response:
     resp.set_cookie('page_size', str(page_size))
     return resp
 
+@APP.api.route('/scraper/')
```

Comment:
Let's go with /decks/updated/. All the other API endpoints are named after the thing they return.

Reply:
Sounds good